main.py

```diff
@@ -1,7 +1,32 @@
 import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
 
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    items = ["Western", "Action", "Dramma"]
```

Comment:
It will be more logical to use "movie_names" or "movie_titles" instead of "items"

Reply:
I think my variant is better, just because i don't need to create 3 variables. Just create one and rewrite it. Memory economy)